### ISSUE TITLE
Proxy not working for some repos

### ISSUE THREAD

**Issue by Phillipus** [archimatetool/archi-modelrepository-plugin#66]
I'm using CCPRoxy with the Archi collaboration server and it doesn't work. I get an "authentication not supported" exception.

However, the same Proxy settings and repo works OK in SmartGit. So there must be something wrong in the ProxyAuthenticater code or somewhere else.

Help needed!

**Comment by cs2ea** [comment]
Hello

I had a lot of trouble connecting to Azure DevOps git repositories from our intranet  via our corporate proxy. Error was usually "Cannot upload git-update-pack", even when populating the http proxy host and credentials in Archi

The push worked in Archi outside of the intranet with the proxy unchecked
The push also worked with  the "Git Extensions" tool from the intranet via the proxy

In other words, the repository was ok and git could reach it via the proxy. 

I resolved the issue in Archi by changing the C:\Program Files\Archi4\jre\lib\net.properties to enable Basic authentication for http proxying & tunnelling and defining the http proxy setting in the collaboration preferences.
_
jdk.http.auth.proxying.disabledSchemes=""
jdk.http.auth.tunneling.disabledSchemes=""
#jdk.http.auth.tunneling.disabledSchemes=Basic
_

Maybe this is similar to the issue you were  facing?

Best Regards 

   Jan D.

**Comment by Phillipus** [comment]
@cs2ea Thanks for the heads up.

I haven't tested Proxy settings for some time now. We currently have this:

`System.setProperty("jdk.http.auth.tunneling.disabledSchemes", "");`

Maybe I need to also set:

`System.setProperty("jdk.http.auth.proxying.disabledSchemes", "");`

Need to test this more...

**Comment by Phillipus** [comment]
`System.setProperty("jdk.http.auth.tunneling.disabledSchemes", "");` has been in since version 0.4.0. Anyone who has a proxy problem, check which version of the plug-in you are using.

**Comment by cs2ea** [comment]
I am perplexed but pleased. I have retested the my whole setup with fresh download of the portable Archi and the latest version of the collaboration plugin (0.5.0). 

As opposed to the behavior I had one month ago (with 0.0.4) I did not need to apply any workaround in net.properties for the push to work fine via our corporate proxy. 

I have no idea what has changed - be it on my side or on the proxy side - but the whole setup now works cleanly with no workarounds. 

Thank you very much for your followup on this topic

Kind Regards
